pkgs/development/haskell-modules/configuration-ghc-9.0.x.nix

```diff
@@ -108,7 +112,11 @@ self: super: {
 
   doctest = dontCheck super.doctest;
 
-  haskell-language-server = throw "haskell-language-server has dropped support for ghc 9.0 in version 2.4.0.0, please use a newer ghc version or an older nixpkgs version";
+  haskell-language-server =
```

Comment:
This one is tricky since it hasn't actually been removed, the package at this point is just broken. Effectively, the `throw` is a bit like `markBroken` here.

Reply:
`markBroken` without aliases is the simplest solution presumably.